inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/BaseSortConf.java

```diff
@@ -29,8 +29,6 @@
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, visible = true, property = "sortType")
 public abstract class BaseSortConf {
 
-    public String sortType;
```

Comment:
This field should not be removed, because it was used by `JsonTypeInfo`.
Suggests set its value in each subclass's constructor.

Reply:
It don't need declare `sortType` field. Annotation will generate sortType data when serialization